app/v6/retrieveForeignPropertyCumulativeSummary/RetrieveForeignPropertyCumulativeSummaryConnector.scala

```diff
@@ -41,12 +41,24 @@ class RetrieveForeignPropertyCumulativeSummaryConnector @Inject() (val http: Htt
     import request.*
     import schema.*
 
+    val queryParams: Seq[(String, String)] =
+      propertyId.map(pid => "propertyId" -> pid.propertyId).toSeq
     val maybeIntent = if (PropertyBusinessFeatureSwitches().isPassIntentEnabled) Some("FOREIGN_PROPERTY") else None
```

Comment:
Can this please be updated. maybeIntent is for tax years where both uk and foreign used the same downstream endpoint so does not apply for tax years 26/27. It can be

`val maybeIntent = Option.when(PropertyBusinessFeatureSwitches().isPassIntentEnabled && taxYear.year <= 2026)("FOREIGN_PROPERTY")` or use other ways.

The stubs should also reflect this.